Source/DOM classes/SVG-DOM/SVGHelperUtilities.m

```diff
@@ -318,20 +318,37 @@ +(CALayer *) newCALayerForPathBasedSVGElement:(SVGElement<SVGTransformable>*) sv
 	
 	[self configureCALayer:_shapeLayer usingElement:svgElement];
 	
+	NSString* actualStroke = [svgElement cascadedValueForStylableProperty:@"stroke"];
+	if (!actualStroke)
+		actualStroke = @"none";
+	NSString* actualStrokeWidth = [svgElement cascadedValueForStylableProperty:@"stroke-width"];
+	
+	CGFloat strokeWidth = 1.0;
+	
+	if (actualStrokeWidth)
+	{
+		SVGRect r = ((SVGSVGElement*) svgElement.viewportElement).viewport;
+		
+		strokeWidth = [[SVGLength svgLengthFromNSString:actualStrokeWidth]
+					   pixelsValueWithDimension: hypot(r.width, r.height)];
+	}
+	
```

Comment:
I moved the stroke width calculation up so that it can be used to calculate the rendered bounding box.